src/applications/personalization/connected-accounts/tests/actions/index.unit.spec.js

```diff
@@ -3,28 +3,12 @@ import sinon from 'sinon';
 import {
   mockFetch,
   resetFetch,
-} from '../../../../../platform/testing/unit/helpers.js';
+  setFetchJSONFailure as setFetchFailure,
```

Comment:
I'm seeing a lot of these renames. What do you think about naming the functions `setFetchFailure` and `setFetchResponse` if that's the name that makes the most sense in a test?

Reply:
Well we have `setFetchJSONResponse` and `setFetchBlobResponse` defined next to each other in the helpers file. So I think having those different, specific names makes sense. I feel less strongly about if we should rename them on import in the consuming test files. I'm happy to kill these renames and use them with their "native" function names if you want.